tools/mode_selector.cwl

```diff
@@ -0,0 +1,39 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: mode_selector 
+doc: "Selects the appropriate input to serve as the output given the mode" 
+requirements:
+  - class: DockerRequirement
+    dockerPull: 'ubuntu:18.04'
+  - class: InlineJavascriptRequirement
+baseCommand: ["/bin/bash", "-c"]
+arguments:
+  - position: 0
+    valueFrom: >-
+      set -eo pipefail
+
+      ${
+          var cmd = " >&2 echo Choosing inputs based on mode;";
+          if (inputs.input_mode == 'WGS' && inputs.wgs_input == null){
+            return "echo WGS run requires wgs_input >&2 && exit 1;"
+          }
+          else if (inputs.input_mode == 'WXS' && inputs.wxs_input == null){
+            return "echo WXS run requires wxs_input >&2 && exit 1;"
+          }
+          return cmd;
+      }
+       
+inputs:
+  input_mode: {type: {type: enum, name: sex, symbols: ["WGS", "WXS"] }, doc: "Select if this run is WGS or WXS"} 
```

Comment:
`name` should probably be `input_mode` here. Using `sex` might still run, but that field name is confusing

Reply:
Good catch. Copy paste goodness.